tools/speckle_connector_loader.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Change the base folder and copy this file to Sketchup Plugins directory
+# If you need to test in several versions of SketchUp, create symlinks to this file
+# ( AppData\Roaming\SketchUp\SketchUp XXXX\SketchUp\Plugins )
+# Create a link to Plugins folder with this command
+#
+# rubocop:disable Layout/LineLength
+#
+# New-Item -ItemType SymbolicLink -Path '~\AppData\Roaming\SketchUp\SketchUp 2022\SketchUp\Plugins\speckle_connector_loader.rb' -Target ~\Documents\Git\Speckle\speckle-sketchup\tools\speckle_connector_loader.rb
+#
+# rubocop:enable Layout/LineLength
+
+SKETCHUP_CONSOLE.show # if you want to show Ruby console on startup
+# base location of your repos - will be merged with specific repos in next step
+home_folder = File.expand_path('~')
+# If you use some other location for your repository, you can change it here
+# but make sure it is not committed as it will change thi setting for all
+# users that use the default setup. Eg:
+
+# Add Speckle folder - uncomment the one you need
+speckle_path = File.join(home_folder, 'Documents', 'Git', 'Speckle', 'speckle-sketchup')
+
+# rubocop:disable Style/GlobalVars
+$LOAD_PATH << speckle_path
+# rubocop:enable Style/GlobalVars
+
+files = %w[speckle_connector]
+
+files.each do |ruby_file|
+  puts "Loading #{ruby_file}"
+  begin
+    require ruby_file
+  rescue LoadError
+    puts "Could not load #{ruby_file}"
+  end
+end
```

Comment:
so there is already documentation in the readme on how to debug with a much simpler `loader.rb` file dropped into the SketchUp plugins dir. i'm not necessarily against updating the recommendations, but if we are to do that then the documentation needs to be updated accordingly. i'd also like to understand what benefits this extra logic gives us when the two-line loader seems to functionally do the same thing?

Reply:
I can see here some improvements can be done to automate deploying plugin etc.. (Maybe it is better to know first how you guys deploying your versions.) But this is separate discussion I believe. But nevertheless I added example of improvements that can be managed by `loader.rb` file. See: https://github.com/specklesystems/speckle-sketchup/pull/58/commits/91f6265b0abce651a640155966fbfa8bf6c1d59f

Reply:
cool i agree that it's worth some discussion! maybe let's remove it form this current PR and open a separate PR with this loader change?

you mention versions - if you're referring to different plugin version for different SketchUp versions then we don't currently do that but do have a need for it! would be keen to hear your thoughts on how that could be done. for context, i'd like to be able to deploy pre and post SketchUp 2022 versions to handle edges differently in 2022 and up using the new entities builder.

Reply:
That's fine by me! Let's exclude this one which is orthogonal to other changes. Let me remove this

Reply:
Removed!

Reply:
amazing thank you! very happy for a new pr to be opened around this and to discuss a bit further 🌟